adept/manager/subproc_env_manager.py

```diff
@@ -15,6 +15,11 @@
 import os
 import pickle
 
+import msgpack
+import msgpack_numpy as m
+
+m.patch()
```

Comment:
can we call this in the constructor so it doesn't get called unless SubProcEnvManager is initialized?